src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,16 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder builder = new StringBuilder();
+        while (value > 0) {
+            int remainder = value % 2;
```

Comment:
good work, let's simplify this solution))
do we need variable `remainder `? try to remove it

Reply:
fixed)